### PR TITLE
NoSearchFilter

### PR DESCRIPTION
**Pull request by praspra2**
# Related Issue

A Pull Request should close a **single** issue; multiple issues can be closed when the issues are **small and related**, but that should be an exception not the rule. Please keep Pull Requests small and targeted; large 'code drops' with dozens of files will be closed and asked to split into reviewable pieces. Reviews that need to be large due to dependencies will be
reviewed on a case-by-case basis.

Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
the issue fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one to aid
in issue tracking.

**NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
page above.

# Description

For all Pull Requests, please describe how the issue was fixed or how the feature was implemented from a summary level. This information will be used to help provide context to the reviewers of the pull request and should be additive to the issue being closed.

- No search results page for dropdown menu
- Jira : [https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-526896]
- (https://app.vidcast.io/share/80bb74c2-a654-4648-ba55-8a7f694f55a3)

# Sample Card

If appropriate, please include a link to a card in one of the samples directories that can be used to validate this change. This can be an existing card or a card added with this PR.

# How Verified

How you verified the fix, including one or all of the following:
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 1]
```diff
@@ -107,22 +107,19 @@ ComboBox {
...
+        enabled: textField.text.length > 0 || comboBox.popup.visible
+        
+        onClicked: {
+			if (textField.text.length > 0) {
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:114]: Indentation needs to be fixed here

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 2]
```diff
@@ -135,35 +132,33 @@ ComboBox {
...
-                comboBox.popup.open();
-                comboBox.filterOptions();
+            comboBox.popup.open();
+            comboBox.filterOptions();
 	
-            }
+                }
            
-        }
+            }
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:161]: Indentation change

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 3]
```diff
@@ -175,18 +170,13 @@ ComboBox {
...
-        }
-
         background: Rectangle {
             x: parent.highlighted ? comboBoxConstants.popoutDelegateCordinates : 0
             y: parent.highlighted ? comboBoxConstants.popoutDelegateCordinates : 0
             width: parent.highlighted ? parent.width - comboBoxConstants.focusRingSize : parent.width
             height: parent.highlighted ? parent.height - comboBoxConstants.focusRingSize : parent.height
             color: comboBoxItemDelegate.pressed ? comboBoxConstants.dropDownElementColorPressed : comboBoxItemDelegate.highlighted ? comboBoxConstants.dropDownElementColorHovered : comboBoxConstants.dropDownElementColorNormal
-            radius: comboBoxConstants.dropDownElementRadius
+            radius: comboBoxConstants.dropDownElementRadius
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:179]: Blank space needs to be removed

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 4]
```diff
@@ -196,29 +186,29 @@ ComboBox {
...
-                comboBox.choiceWidth = Math.max(maxWidth, comboBox.choiceWidth);
+    contentItem: Text {
+        text: modelData.text
+        font.family: "Segeo UI"
+        font.pixelSize:  inputFieldConstants.pixelSize
+        color: inputFieldConstants.textColor
+        elide: Text.ElideRight
+        onImplicitWidthChanged: {
+            var maxWidth = implicitWidth > comboBoxConstants.maxDropDownWidth ? comboBoxConstants.maxDropDownWidth : implicitWidth;
+            comboBox.choiceWidth = Math.max(maxWidth, comboBox.choiceWidth);
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:197]: Add indentation

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 5]
```diff
@@ -196,29 +186,29 @@ ComboBox {
...
 
-	onClicked: {
-                    comboBox.currentIndex = index
-                    comboBox.popup.close()
-                }
-	}
+	    onClicked: {
+            comboBox.currentIndex = index
+            comboBox.popup.close()
+            }
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:204]: Add indentation

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 6]
```diff
@@ -196,29 +186,29 @@ ComboBox {
...
+	    }
     
 
     popup: Popup {
         y: comboBox.height + 5
         width: Math.max(comboBox.choiceWidth, comboBox.width)
-        padding: comboBoxConstants.dropDownPadding
+        padding: comboBoxConstants.dropDownPadding
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:211]: Remove blank space

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 7]
```diff
@@ -231,43 +221,53 @@ ComboBox {
         }
 
         background: Rectangle {
-            anchors.fill: parent
+            height: comboBox.delegateModel.count === 0 ? comboBoxConstants.dropDownHeight : comboBoxListView.height + 16
             color: comboBoxConstants.dropDownBackgroundColor
-            border.color: comboBoxConstants.dropDownBorderColor
+            border.color: comboBoxConstants.dropDownBorderColor
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:226]: Remove blank space

File: source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml [hunk 8]
```diff
@@ -231,43 +221,53 @@ ComboBox {
...
+		}
+	}
 
     background: Rectangle {
         radius: inputFieldConstants.borderRadius
         color: inputFieldConstants.backgroundColorNormal
         border.color: inputFieldConstants.borderColorNormal
         border.width: inputFieldConstants.borderWidth
-    }
+    }
```
Comment (snbehar) [inline source/qml/Samples/QmlVisualizer/AdaptiveCardQml/FilteredChoiceSetRender.qml:270]: Remove blank space

**Review by snbehar — APPROVED** [review]
LGTM!

**Review by ramkrishnajana — APPROVED** [review]

**Pull request merged**